KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -418,6 +418,7 @@ public enum UITexts {
         static let freeCancellationASAP = "Text.Quote.FreeCancellationASAP".localized
         static let freeCancellationAndKeyword = "Text.Quote.FreeCancellationAndKeyword".localized
         static let freeCancellationBeforeDriverEnRoute = "Text.Quote.FreeCancellationBeforeDriverEnRoute".localized
+        static let feesAndTaxesIncluded = "Text.Quite.FeesAndTaxesIncluded".localized
```

Comment:
Could you update the value to "Text.Quote.FeesAndTaxesIncluded"? When I got the strings from Phrase I only added this key in the Localizable.strings files, not in UITexts.